data/json/snippets/photos.json

```
@@ -139,7 +139,23 @@
-      }
+      },
+      {
+        "id": "wallet_picture_35",
+        "text": "A blurry photo tooken in the bright daylight.  a dog sits in the middle of the frame with a ball in it's mouth, it's fur completely caked in mud.  Two kids are seen to the left playing in a nearby pond, Most likely the cause of the dogs soiled fur.  On the back, a message reads: \"You had a long run.  I still miss you, always.\""
+      },
+      {
+       "id": "wallet_picture_36",
+       "text": "This photograph demonstrates what happens when you let too many puppies into a room with one person.  The man in the middle of the dogpile is absolutely overjoyed, as he's covered in canines all shapes and sizes.  A note on the back says \"Winner of the company lottery.\""
+      },
+      {
+       "id": "wallet_picture_37",
+      "text": "A simple photo of two children hugging.  The photo is awfully thick though, and a tap seems to displace multiple other photos that were sticking together.  Each consecutive photo shows the two in later stages of their life.  The last photo has the two wearing a pair of graduation caps and robes, In a similar hugging position as the first." 
+      },
+      {
+       "id": "wallet_picture_38",
+       "text": "A strange gray fruit is placed on a pedesteal, like a relic would be placed apon a stand, for others to admire.  Clearly it isn't anything originating from this world, and the vein-like tendrils along it's skin only convince you of that even more.  Deep scratches have been used to carve out the message \"I'm sorryI'm sorryIm sorr'm soryI'm sorrm sorrsorr\" on the back, In increasingly less legible handwriting.  It gives off the faint smell of mold."
```

Good photo, but in general, I’d be cautious about making clearly cataclysmic entities too bluntly plane in these pictures. I know that’s somewhat rich considering that picture 34, well, exists, but there it’s evident that it was taken hurriedly and covertly printed when shit had already hit the fan. I would tentatively suggest giving the same treatment to this one, implying that the photo was rushed, printed badly, or otherwise not professionally done.
Also, “deep scratches on the back.” On the back of what, exactly, the fruit or the picture? If the former, how is the viewer seeing it, and if it’s the latter, that’s got to be some dummy thick paper?
```suggestion
       "text": "A strange grey fruit is placed on a pedestal, like a relic would be placed on a stand for others to admire.  Clearly, it isn't anything originating from this world, and the vein-like tendrils along its skin only convince you of that even more.  Deep scratches have been used to carve out the message \"I'm sorry I'm sorry Im sorr'm soryI'm sorrm sorrsorr\" on the back, in increasingly illegible handwriting. It gives off the faint smell of mould."
```